gponty/symfony-monday-api-bundle/1.0/config/packages/monday.yaml

```
@@ -0,0 +1,3 @@
+# Read the documentation: https://github.com/gponty/symfony-monday-api/blob/main/README.md
+monday:
+    api_key: '%env(resolve:MONDAY_API_KEY)%'
```

As the API key is a scalar, there is no need to "resolve", isn't it?

```suggestion
    api_key: '%env(MONDAY_API_KEY)%'
```